src/pages/CheckoutDetails.jsx

```
@@ -0,0 +1,193 @@
+          {row.map((cell, cellIndex) => (
+            <td key={`cell-${rowIndex}-${cellIndex}`}>{cell}</td>
+          ))}
+        </tr>
+      ))}
+      <tr>
+        <td colSpan={3}>Total</td>
+        <td>{summary?.Rate}</td>
+        <td>{summary?.RoomTax}</td>
+        <td>{summary?.Plan}</td>
+        <td>{summary?.FoodBill}</td>
+        <td>{summary?.PostBill}</td>
```

ESLint

[error] 'summary.PostBill' is missing in props validation

(react/prop-types)